client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -102,13 +107,18 @@ module.exports = React.createClass( {
 			statusText = this.getStatusText(),
 			realtiveTimeClass = ( timeText ) ? 'post-relative-time-status' : null,
 			innerText = ( <span>{ timeText }{ statusText }</span> ),
+			time = this.getTimestamp(),
 			details;
 
 		if ( this.props.link ) {
```

Comment:
Great idea. Was thinking about it too, but didn't want to put it in this PR

Thanks for the merge